examples/sdkservice.go

```diff
@@ -1,13 +1,13 @@
 package main
 
 import (
-
+	"fmt"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/fakeservice"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/sdkprovider"
-	"fmt"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 )
 
 func getFakeService(ip, username, password, apiVersion string, synchronous bool) (sdkprovider.NsGroupService, error) {
```

Comment:
Per my earlier comment, the fakeservice package should be kept in sync with the service changes.  Converting this example to the non-deprecated function should clarify it.